canonical/snapcraft.io
# Convert brand store members to TypeScript

## Done
Converted brand store members to TypeScript

## How to QA
- Go to https://snapcraft-io-4310.demos.haus/admin
- Go to a store where you are an admin
- Check that you can make changes to members roles

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-4852
